.swift-format-version

```diff
@@ -0,0 +1 @@
+0.50500.0
```

Comment:
As I mentioned in a comment, let's vendor the tool and prevent developers from having to mess up with version management of their environment's tools.

Reply:
it is vendored - I thought we were using this to make it easier to spot what version you are using, I can delete it. However, there is a big drawback I realized when fixing the CI. The vendored binary has an absolute path pointing to a SwiftSyntax library in Xcode.

If you build `swift-format`, you'll see it when you run `otool -l swift-format | grep Xcode`. You can add another path with `install_name_tool -add_rpath /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx swift-format` but it will still depend how you install Xcodes. There are three options going forward:
- live with this limitation and document it (users would have to have locally Xcode at `/Applications/Xcode.app` or at least an alias
- do not vendor `swift-format`
- close this PR and stick to `swiftformat`

Reply:
Can't we just have `fourier` run `install_name_tool` using the path from `xcode-select`?

Reply:
> Can't we just have fourier run install_name_tool using the path from xcode-select?

That's an excellent idea. Yes, we could.